src/signals/incident/components/form/ContainerSelect/components/ContainerEditList/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback } from 'react';
+import type { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import { themeSpacing } from '@amsterdam/asc-ui';
+import { Close } from '@amsterdam/asc-assets';
+
+import Button from 'components/Button';
+import type { FeatureType, Item } from '../../types';
+
+const DEFAULT_ICON_SIZE = 40;
+const REMOVE_BUTTON_SIZE = 32;
+const REMOVE_ICON_SIZE = 12;
+const NAME = 'containerEditList';
+
+const StyledButton = styled(Button)`
+  flex-shrink: 0;
+  box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.1);
+`;
+
+const List = styled.ul`
+  padding: 0;
+`;
+
+const ListItem = styled.li`
+  line-height: ${themeSpacing(4)};
+  padding: ${themeSpacing(1, 0)};
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: initial;
+
+  &:focus {
+    outline-style: none;
+  }
+`;
+
+const StyledIcon = styled.span<{ url: string }>`
+  display: inline-block;
+  margin-right: ${themeSpacing(2)};
+  flex-shrink: 0;
+  width: ${DEFAULT_ICON_SIZE}px;
+  height: ${DEFAULT_ICON_SIZE}px;
+  background-image: url(${({ url }) => url});
+  background-size: cover;
+`;
+
+const StyledItemSpan = styled.span`
+  display: flex;
+  align-items: center;
+  margin-right: ${themeSpacing(2)};
+`;
+
+export interface ContainerEditListProps {
+  selection: Item[];
+  onRemove: (id: string) => void;
+  featureTypes: FeatureType[];
+  className?: string;
+}
+
+const ContainerEditList: FunctionComponent<ContainerEditListProps> = ({
```

Comment:
There is a lot of overlap with `ContainerList` in this component. Lets merge the two. When onRemove is not passed then we don't show the `Close` icon.